exercises/F-logical-operators/exercise2.js

```diff
@@ -5,7 +5,27 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(number1) {
+  let result = number1 < 0;
+  return result;
+}
+
+function isBetween5and10(number1) {
+  let result = number1 <= 10 && number1 > 5;
+  return result;
+}
+
+function isShortName(name1) {
+  let result = name1.length;
+  if (result < 10) return true;
+  else return false;
+}
+
+function startsWithD(name1) {
+  if (name1.charAt(0) === "D") {
+    return true;
+  } else return false;
+}
```

Comment:
I would suggest for `if... else....` blocks of code that you continue to use the curly braces and indentation even when you only have one statement inside the braces. This makes it very clear what gets run in each condition and it's easier to spot mistakes. 

E.g.
```
if (result < 10) {
    return true;
} else {
    return false;
}
```
It is accepted to miss out the braces if you have only one statement being run, but in that case you should add indentation.
E.g.
```
if (result < 10)
    return true;
else
    return false;
```

Reply:
Oh I see. I thought it looks more professional this way :D,. Obviously it's not, I'll change the code, thank you